article/2021/Web-Performance-for-Product-Managers.md

```diff
@@ -2,179 +2,178 @@
-- improving the 'critical path' or render blocking, and
-- using directives to preload assets or load scripts asynchronously.
+- JavaScript，
+- 改善“关键路径”或渲染遮挡，以及
+- 使用指令来预加载页面元素或异步加载脚本。
 
-While the focus in most cases tends to be directed toward front-end developers, it's important to remember the back-end as well if you see higher than normal start render times or increases in more basic metrics like time to first byte.
+尽管大多数情况下，产品经理的注意力都都集中在前端开发者上，但是，如果你察觉到了过长的开始渲染时间，或更多不正常的基本指标（例如到第一个字节的时间）增加，也要关注后端开发工作。这一点非常重要。
 
-# Images
+### 图像
 
-Image optimization continues to be one of the areas where huge performance gains can be made, often with little effort. Image compression has continued to evolve. New formats have been introduced that provide a rich feature set while minimizing bandwidth. In some cases these formats are specific to the browser, which can add some complexity to your workflow.
+图像优化仍然是可以以最小付出换取巨大性能提升的领域之一。随着图像压缩技术的不断发展，新的图像格式被引入实际运用当中。这些格式提供了丰富的功能集，同时将所需的带宽最小化。在某些情况下，这些格式是特定于浏览器的，有可能导致工作流程的复杂性增加。
 
-Regardless of whether or not you have the opportunity to make an impact in this area, understanding your image pipeline – whether it's in house or managed as a service – is a plus. This is especially true of sites that maintain a large product catalog or include a fair amount of user-generated content.
+无论你是否有机会实际参与这一领域的工作，理解图像管道（不管是在内部还是作为服务进行管理）都是大有裨益的。对于维护大型产品目录或包含大量用户生成内容的网站，尤其如此。
 
-It's also important to note that, while not all performance gains will show up in your metrics, users with poor bandwidth or restrictive data plans will thank you. Consider setting your [performance budgets](https://speedcurve.com/blog/performance-budgets-guide/) related to images on size and number of requests as opposed to duration.
+还需要注意的是，虽然并非所有性能提升都会显示在您的指标中，但是这些工作对于带宽不足或数据计划受限的用户将是十分令人感激的。考虑设置与图像的数量或大小相关的[性能预算](https://speedcurve.com/blog/performance-budgets-guide/)，而不仅考虑持续的时间。
 
-# Networks
+### 网络
 
-Yeah. The internet. It's consistently inconsistent. How things get from point A to point B still amazes me when you think about the complexity involved in sending bits over the wire.
+正是如此。互联网一直以来都是反常的。通过线缆，从A点到B点传输一又一个比特————当我想到这件事本身所涉及的复杂性时，我仍然感到惊讶与激动。
 
-For your purposes, you might be interested in how your technology partners (which you pay good money for) impact your performance. Content Delivery Networks (CDN) like Akamai, Fastly, Cloudflare, Cloudfront can sometimes have a big impact on performance. This traditionally manifests more in back-end time, but can have a huge impact on front-end as well when you factor in asset delivery (images, static content) as well as some advanced optimizations available at the edge. These platforms also give you the power to make things worse, not better, if you aren't careful.
+对你来讲，你可能会对技术合作伙伴如何增进服务的性能感到好奇，毕竟你为此付出了一大笔钱。诸如 Akamai，Fastly，Cloudflare，Cloudfront 之类的内容交付网络（CDN）有时会对性能产生重大影响。传统上，这些影响在服务后端时间表现得更多，但是当考虑到从服务端到 Web 的一些资源交付（图像或其他静态内容）以及边缘可用的一些高级优化时，也会对前端产生巨大影响。如果你不多加留意，这些平台只能使情况变得更糟，而不是变得更好。
```

Comment:
d for deliver not deployment
```suggestion
对你来讲，你可能会对技术合作伙伴如何增进服务的性能感到好奇，毕竟你为此付出了一大笔钱。诸如 Akamai、Fastly、Cloudflare 和 Cloudfront 之类的内容分发网络（CDN）有时会对性能产生重大影响。传统上，这些影响在服务后端时间表现得更多，但是当考虑到从服务端到 Web 的一些资源分发（图像或其他静态内容）以及边缘可用的一些高级优化时，也会对前端产生巨大影响。如果你不多加留意，这些平台只能使情况变得更糟，而不是变得更好。
```

Reply:
[https://en.wikipedia.org/wiki/Content_delivery_network](url)